packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -192,14 +192,18 @@ export default class MeetingInfoV2 {
 
     const options = {
       method: HTTP_VERBS.POST,
-      service: 'webex-appapi-service',
-      resource: 'meetingInfo',
       body
     };
 
     const directURI = await MeetingInfoUtil.getDirectMeetingInfoURI(destinationType);
 
-    if (directURI) options.directURI = directURI;
+    if (directURI) {
+      options.uri = directURI;
+    }
+    else {
+      options.service = 'webex-appapi-service';
```

Comment:
Looks like `'webex-appapi-service'` used in many places we should have constant for this probably in `packages/node_modules/@webex/plugin-meetings/src/constants.ts`

Reply:
I've changed this to be a constant